lib/index.js

```diff
@@ -113,9 +114,10 @@ export const reduceChain = (state, action) => {
         if (numBlocks && blocks.length >= numBlocks) {
           newBlocks.pop();
         }
-        newBlocks = newBlocks.map(block =>
+        newBlocks = newBlocks.map(block => {
           block.depth = block.depth ? block.depth + 1 : 1;
```

Comment:
should be actual block data. Line 119 returns the whole block with depth updated. This was to avoid querying all of the recent blocks when we get a new block just to update their depth. i.e. we get a new block event with the new block data, add that to our list of blocks, but the previous blocks still have their old depth even though they've all been incremented by one.